backend/app/Http/Controllers/QuizLogController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\QuizLog;
+use App\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Validation\Rule;
+use Illuminate\Support\Facades\Auth;
+
+class QuizLogController extends Controller {
+    public function index() {
+        return $this->getUserWithQuizLogs(Auth::id());
+    }
+
+    public function store(Request $request) {
+        $id = Auth::id();
+        $request->merge(['user_id' => $id]);
+
+        $attrs = $this->validateQuizLog($request);
+        $quizlog = QuizLog::create($attrs);
+        return $this->getUserWithQuizLogs($id);
```

Comment:
Please put new line before `return`
```suggestion
        $quizlog = QuizLog::create($attrs);
        
        return $this->getUserWithQuizLogs($id);
```